### PR TITLE
Dependency update: Bump @truffle/error and @types/node

### PR DESCRIPTION
**Pull request by eggplantzzz**

### CODE REVIEW

File: packages/compile-common/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "types": "dist/src/index.d.ts",
   "dependencies": {
-    "@truffle/error": "^0.1.0",
+    "@truffle/error": "0.1.0",
```
Comment (haltman-at) [inline packages/compile-common/package.json:24]: Is there some reason you pinned these...?  Our dependencies on our own packages aren't normally pinned.

File: packages/plugins/package.json [hunk 2]
```diff
@@ -33,9 +35,7 @@
     "ts-jest": "^26.1.0",
     "typescript": "^4.1.4"
   },
-  "dependencies": {
-    "@truffle/error": "^0.0.8",
```
Comment (haltman-at) [inline packages/plugins/package.json:37]: Yikes, thanks for catching this!

**Review by haltman-at — COMMENTED** [review]
Thanks for catching that outdated `@truffle/error` dependency!  This doesn't seem to involve `@types/node` at all though as best I can tell?

File: packages/compile-common/package.json (hunk 1 above)
Comment (eggplantzzz) [reply packages/compile-common/package.json:24]: yeah we should probably `^` this - sorry, `lernaupdate` pins it by default and I neglected to doctor it up

**Pull request merged**